games-roguelike/tome/tome-2.3.5.ebuild

```diff
@@ -1,8 +1,8 @@
 # Copyright 1999-2016 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
-inherit eutils games
+EAPI=7
+inherit multilib
```

Comment:
Is anything using multilib.eclass here? get_libdir is commonly the reason but that's [part of](https://blogs.gentoo.org/mgorny/2015/11/13/the-ultimate-guide-to-eapi-6/) >= EAPI 6

Reply:
Didn't realize it was builtin after EAPI 6, that's the reason I added it.

Reply:
Now you know! (This is quite a common thing to have to fix, actually)